### ISSUE TITLE
branch: description-temp

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#181]
A branch with initial commits for #180. Should be short lived likely merged with master quickly. Assigning to @mbarlow12 just so he is aware since he is working on that issue.

**Comment by ariel-phet** [comment]
Seems like this can be closed.

**Comment by jessegreenberg** [comment]
This issue is to document an existing branch which still exists. We should check the branch against master to see if it can be deleted, and then close this issue.

**Comment by jessegreenberg** [comment]
This wasn't merged into master, but there is a lot of description work that may be useful to keep for #180, when we pick that up again this should be reviewed.

**Comment by jessegreenberg** [comment]
For reference, these are the commits on description-temp:
```
$ git log master..description-temp
commit ac4e81a4c527ad40f2ef5e3f00457f61e0d4ec49 (origin/description-temp, description-temp)
Merge: 507e8c6 9b0297b
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Thu May 31 18:52:26 2018 -0600

    Merge branch 'master' into description-temp

commit 507e8c6054b506cc2d2a1f392644ad86d5d86e73
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Fri May 18 17:26:39 2018 -0600

    removed comment, slider alerts draft see https://github.com/phetsims/molecules-and-light/issues/180

commit e80e0c160639d3a26b20281cb1d3633d084bbe27
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Fri May 18 11:08:13 2018 -0600

    SpectrumWindowDialog -> LightSpectrumDialog, see https://github.com/phetsims/molecules-and-light/issues/186

commit 02703334ffbfe96c23cd6b6f2b493e4602666773
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Fri May 18 11:03:41 2018 -0600

    completed molecule and photon interaction alerts, see https://github.com/phetsims/molecules-and-light/issues/180

commit 185fe63de25ba8ce0ab6aee2d84d4b3b15f384be
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Thu May 17 15:53:27 2018 -0600

    alert stubs added, TODO: use a11y string pattern/file and refactor unnecessary code, see https://github.com/phetsims/molecules-and-light/issues/180

commit a49bf01799b4b9edf93c3ac3cb3e2c3a6a8d32e1
Merge: 1b07398 55323b4
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Tue May 1 08:48:11 2018 -0600

    2nd merge from master

commit 1b07398ed89f329a5af7dcfbaccc78d6596d3768
Author: Michael Barlow <m.barlow12@gmail.com>
Date:   Tue May 1 08:47:20 2018 -0600

    merge changes from master
```

**Comment by jessegreenberg** [comment]
This branch can be deleted and closed now.

**Comment by jessegreenberg** [comment]
Deleted.

**Comment by jessegreenberg** [comment]
Re-deleted as part of https://github.com/phetsims/special-ops/issues/198
